### DISCUSSION TITLE
Guitar Classic - A Game Inspired by Harmonix Era Guitar Hero (1 & 2)

### DISCUSSION THREAD

**Discussion by dreammixgames** [melanchall/drywetmidi#153] [Show and tell]
Hey DryWetMidi creators, and others who may come across this. Here's a game that I'm working on called Guitar Classic (using DWM). I'm very thankful to the creator of this parser, as it has enabled me to make something that I've always wanted to!

https://youtu.be/Lpe-VoP9pck

**Comment by melanchall** [comment]
Hi,

Thanks for using the library and good luck with your project! Feel free to ask me any questions.

**Reply by dreammixgames** [reply]
Thank you! :)
